.github/workflows/minor_version_release.yml

```diff
@@ -25,17 +21,47 @@ jobs:
           python -m pip install --upgrade pip
           pip install pydoc-markdown==4.5.1
 
-      - name: Convert version name to regular semver (e.g. v1.10.x -> v1.10.0)
-        id: regular_semver
-        env:
-          version: ${{github.event.inputs.version}}
+      - name: Get current version
+        id: current-version
+        shell: bash
+        # We only need `major.minor` in Readme so we cut the full version string to the first two tokens
+        run: echo "::set-output name=minor::$(cat VERSION.txt | cut -d "." -f 1,2)"
+
+      - name: Create new version branch
+        run: |
+          git checkout -b v${{ steps.current-version.outputs.minor }}.x
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git push -u origin v${{ steps.current-version.outputs.minor }}.x
+
+      - name: Bump version on main
+        id: bump-version
+        run: |
+          git fetch
+          git checkout main
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git pull
+          python ./.github/utils/bump_version.py
```

Comment:
You can get rid of the Python script with this line:
```
cat VERSION.txt | awk -F. '/[0-9]+\./{$2++;print}' OFS=. | tee > VERSION.txt
```

Reply:
Sure